ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -149,6 +149,7 @@ export default Route.extend({
     let backend = this.enginePathParam();
     let backendModel = this.store.peekRecord('secret-engine', backend);
     let has404 = this.has404;
+    let capabilities = this.store.findRecord('capabilities', `${backend}/data/${secret}`);
```

Comment:
TODO: this might have been a part of trying to hide the "create secret work" I believe I can remove this but want to double check.